modules/sdk/src/main/scala/org/tessellation/sdk/cli/StateChannelWhitelist.scala

```diff
@@ -0,0 +1,31 @@
+package org.tessellation.sdk.cli
+
+import cats.data.NonEmptySet
+
+import scala.io.Source
+
+import org.tessellation.schema.address.Address
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.sdk.config.AppEnvironment
+import org.tessellation.sdk.config.AppEnvironment._
+
+import io.circe.parser.decode
+
+object StateChannelWhitelist {
+
+  def get(env: AppEnvironment): Option[Map[Address, NonEmptySet[PeerId]]] =
+    env match {
+      case Dev     => None
+      case Testnet => Some(loadTestnet())
```

Comment:
I think the seedlist should be hardcoded and in the future parametrized by ordinal. So I wouldn't load it from a file.